examples/text_classification/pretrained_models/deploy/serving/client.py

```diff
@@ -13,94 +13,25 @@
 # limitations under the License.
 
 import argparse
-import time
 import numpy as np
-import os
 
 import paddle
-from paddlenlp.data import Stack, Tuple, Pad
+from paddlenlp.data import Tuple, Pad
 from paddlenlp.transformers import ErnieTinyTokenizer
 from paddle_serving_client import Client
 from scipy.special import softmax
 
+from ...utils import convert_example
```

Comment:
from ...utils import convert_example -> from utils import convert_example 

Because we run the script from the parent directory by
`python deploy/python/predict.py --model_dir=./export`

Reply:
Done.